lua/package-info/ui/generic/loading-status.lua

```diff
@@ -28,10 +33,22 @@ M.new = function(message)
         id = math.random(),
         message = message,
         is_ready = false,
+        notification = nil,
     }
 
+    if nvim_notify then
+        instance.notification = vim.notify(message, vim.log.levels.INFO, {
+            title = title,
+            icon = SPINNERS[1],
+            timeout = false,
```

Comment:
I'm curious whether this timeout is reasonable. Should there be an option to modify the notification timeout?